### ISSUE TITLE
AtlDotnet renders the audiofile unusable

### ISSUE THREAD

**Issue by JKamsker** [Zeugma440/atldotnet#125]
## The problem

Opening a track and immediately saving it renders the file unusable

FFmpeg cant read it
Vlc cant read it
![image](https://user-images.githubusercontent.com/11245306/149244099-91e01d5e-d01f-4df3-a0a2-0132879b61bb.png)

## Environment

* ATL version (or git revision) that exhibits the issue: 4.20.
* Last ATL version that did not exhibit the issue (if applicable): unknown, gotta test
* OS/version used to run ATL: Win 10

## Details

No log is produced

## Code To Reproduce Issue [ Good To Have ]

```
var newName = @"audiobook - Kopie.m4b";
var track = new Track(newName);
track.Save();
```

**Comment by JKamsker** [comment]
Have tested older versions, 4.1& 4.0 also show this behaviour.

**Comment by Zeugma440** [comment]
Fixed in today's v4.03
